### PR TITLE
faktorial

### PR DESCRIPTION
**Pull request by kzveg**

### CODE REVIEW

File: demo.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+body{
+    margin 0;
+    width: 100vw;
+    height: 100vh;
+    background-color: rgba(139, 4, 4, 0.986);
+}
+
+.first {
+    display: flex;
+    flex-direction: column;
```
Comment (RomanBorysenko) [inline demo.css:10]: use 'row' instead of 'column'

File: demo.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+.box1 div {
+    border: 5px solid whitesmoke;
+    height: 100px;
+    width: 100px;
+    margin: 8px;
+    margin-top: 15px;
+}
+
+.box2 div {
```
Comment (RomanBorysenko) [inline demo.css:33]: you don't need duplicate the same classes

File: demo.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
+body{
```
Comment (RomanBorysenko) [inline demo.css:1]: The contents of the page are not centered

**Review by RomanBorysenko — COMMENTED** [review]
why do you have 2 commits here?

File: demo.css (hunk 1 above)
Comment (kzveg) [reply demo.css]: fixed

File: demo.css (hunk 3 above)
Comment (kzveg) [reply demo.css:1]: fixed

File: demo.css (hunk 2 above)
Comment (kzveg) [reply demo.css]: fixed

**Pull request merged**